compiler/qsc_frontend/src/resolve/tests.rs

```diff
@@ -513,21 +513,19 @@ fn ambiguous_prelude() {
         }
         "},
         &expect![[r#"
-            namespace namespace3 {
+            namespace namespace8 {
                 function item1() : Unit {}
             }
 
-            namespace namespace4 {
+            namespace namespace5 {
                 function item3() : Unit {}
             }
 
-            namespace namespace7 {
+            namespace namespace9 {
                 function item5() : Unit {
-                    A();
+                    item3();
                 }
             }
-
-            // AmbiguousPrelude { name: "A", candidate_a: "Microsoft.Quantum.Canon", candidate_b: "Microsoft.Quantum.Core", span: Span { lo: 181, hi: 182 } }
```

Comment:
If this test no longer produces an Ambiguous Prelude error, then the test is broken, as that was the purpose of the test. If the intended result is for this to not produce Ambiguous Prelude errors anymore, then maybe the test should be removed?

Reply:
I think I see why... this used to be ambiguous prelude because `Microsoft.Quantum.Canon` was in the prelude, but now only `Std.Canon` is in the prelude. To keep the test checking what it is supposed to check, it should have the first namespace updated the same way.

Reply:
Yep, good catch -- updated!